mono/metadata/threads.c

```diff
@@ -1873,15 +1873,16 @@ ves_icall_System_Threading_Thread_GetPriority (MonoThreadObjectHandle this_obj,
  *
  * Sets the priority of the given thread.
  */
+ICALL_EXPORT
 void
-ves_icall_System_Threading_Thread_SetPriority (MonoThreadObjectHandle this_obj, int priority, MonoError *error)
+ves_icall_System_Threading_Thread_SetPriority (MonoThreadObjectHandle this_obj, int/*MonoThreadPriority FIXMEcxx*/ priority, MonoError *error)
```

Comment:
Remove this unreadable comment.

Reply:
Move it? There are a few places where we have ints that are really enums.